src/app/core/common-components/entity-form/entity-form.service.ts

```diff
@@ -147,7 +199,9 @@ export class EntityFormService {
       this.subscriptions.push(statusChangesSubscription);
     }
 
-    this.defaultValueService.handle(group, entity);
+    // if (handleDefaultValues) {
+    //   this.defaultValueService.handle(group, entity);
+    // }
```

Comment:
is this parameter obsolete or needs to be implemented still?

Reply:
This method will be replaced by createExtendedEntityForm after this refactoring, so this will be removed